content/documentation/staging/configuration/authentication/openid.adoc

```diff
@@ -274,3 +274,11 @@ spec:
       http_proxy: http://USERNAME:PASSWORD@10.0.1.1:8080/
       https_proxy: https://USERNAME:PASSWORD@10.0.0.1:8080/
 ----
+
+=== Using with Keycloak
+
+When using OpenId with Keycloak, you will need to enable the `Standard Flow Enabled` and `Implicit Flow Enabled` options on the Client (in the Administration Console):
+
+image::/images/documentation/authentication/keycloak-implicit-client.png[Client configuration screen on Keycloak]
+
+Without those, you will get an error like `Client is not allowed to initiate browser login with given response_type. Implicit flow is disabled for the client.`.
```

Comment:
Yes, but no.

In the `Requirements` section, the first paragraph says:

```
If you want to enable usage of the OpenId’s authorization code flow, make sure that the Kiali’s signing key is 16, 24 or 32 byte long. If you setup a signing key of a different size, Kiali will only be capable of using the implicit flow.
```

So, it depends on user config. If the user configures a signing key of any of the mentioned lengths, the `Implicit Flow Enabled` option is not really required. Moreover, its safer to leave it disabled, but it depends on user needs.

I think this paragraph will be confusing to some people. Because in the _Introduction_ section it's said that `Kiali supports the authorization code flow (preferred) and the implicit flow`, and IMO this paragraph is implicitly telling users that Kiali needs "Implicit flow" for auth to succeed :/